ui/StatusQ/src/StatusQ/Popups/StatusMenuInstantiator.qml

```diff
@@ -5,7 +5,7 @@ import QtQml.Models 2.15
 Instantiator {
     id: root
 
-    property Menu menu
+    property var menu
```

Comment:
This was failing the SanityChecker tests

Reply:
I added a fix for it already!! its also landed in master. Adding import import QtQuick.Controls.Universal 2.15solves the issue of modal not showing up. Not sure what sanity checker was complaining about. Can you please rebase and check without this change?

Reply:
Yeap agree, but because we added in StatusMenu, it created a type mismatch when not added here. SO we want both this and the StatusMenu to inherit from same Menu, if that makes sense